WellenceChallenge/WellenceApp/services.py

```diff
@@ -0,0 +1,43 @@
+from .models import Tasks
+from django.shortcuts import get_object_or_404
+
+class TaskService:
+    @staticmethod
+    def create_task(task_data):
+        # Handle the is_urgent field
+        is_urgent = task_data.get('is_urgent')
+        task_data['is_urgent'] = is_urgent in [True, 'on', 'true', 'True', 1] # Convert to boolean
+
+        # Get the latest id and increment it
+        latest_id = Tasks.objects.latest('id').id if Tasks.objects.exists() else 0
+        new_id = latest_id + 1
+        task_data['id'] = new_id
+
+        # Create a new Tasks object
+        task = Tasks.objects.create(**task_data)
+        return task
+
+    @staticmethod
+    def get_all_tasks():
```

Comment:
Similarly, specify what you are returning.
`def get_all_tasks() -> QuerySet:` is better than nothing though you should ideally not return QuerySets in services.